lib/src/presentation/code_text_view.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:highlights_plugin/highlights_plugin.dart';
+import 'package:highlights_plugin/model/code_highlight.dart';
+import 'package:kode_view/src/presentation/styles/text_styles.dart';
+import 'package:kode_view/src/presentation/text_selection_options.dart';
+import 'package:kode_view/src/utils/extensions/collection_extensions.dart';
+import 'package:kode_view/src/utils/extensions/text_extensions.dart';
+
+class CodeTextView extends StatelessWidget {
+  const CodeTextView({
+    super.key,
+    required this.code,
+    this.maxLines,
+    this.options,
+    this.showCursor,
+    this.onTap,
+    this.language,
+    this.theme,
+  });
+
+  final splitter = const LineSplitter();
+  final String code;
+  final int? maxLines;
+  final bool? showCursor;
+  final TextSelectionOptions? options;
+  final GestureTapCallback? onTap;
+
+  final String? language;
+  final String? theme;
+
+  const CodeTextView.preview(
+      {super.key,
+      required this.code,
+      this.options,
+      this.onTap,
+      this.showCursor,
+      this.language,
+      this.theme})
```

Comment:
Add trailing commas to match with:
https://dart.dev/tools/linter-rules/require_trailing_commas